### DISCUSSION TITLE
Input link modal in editor disappear after click at second time or later

### DISCUSSION THREAD

**Discussion by bachvtuan** [wintercms/winter#1005] [Q&A] [answered]
https://github.com/wintercms/winter/assets/2065400/65ac3e64-91ec-4bf1-abd5-4cd38e02d1a2

I have updated to the latest WinterCMS version 1.2.3. The first time is ok but it won't let input input link in the next time, the modal just get closed after showing like 1 second.

**Comment by bachvtuan** [comment]
I have installed WinterCMS page plugin version 2.1.2

**Comment by bachvtuan** [comment]
This is bug from Page plugin, I will close this discussion and report bug to Page plugin repo.

**Comment by LukeTowers** [comment, marked as answer]
This was fixed by https://github.com/wintercms/winter/commit/23ba829f19e6062932f61efb390e36c91b8bc497, awaiting release in 1.2.4
